data/json/materials.json

```diff
@@ -1062,6 +1125,125 @@
       { "fuel": 2, "smoke": 1, "burn": 2, "volume_per_turn": "2500_ml" },
       { "fuel": 2, "smoke": 1, "burn": 3 }
     ]
+  },
+  {
+    "type": "material",
+    "ident": "dried_vegetable",
+    "name": "Dried vegetable",
+    "density": 4,
+    "specific_heat_liquid": 2.5,
+    "specific_heat_solid": 3.5,
+    "latent_heat": 100,
+    "edible": true,
+    "bash_resist": 1,
+    "cut_resist": 1,
+    "acid_resist": 2,
+    "fire_resist": 1,
+    "elec_resist": 1,
+    "chip_resist": 1,
+    "dmg_adj": [ "lightly damaged", "damaged", "very damaged", "thoroughly damaged" ],
+    "bash_dmg_verb": "bruised",
+    "cut_dmg_verb": "cut",
+    "vitamins": [ [ "vitC", 2 ] ],
+    "burn_data": [
+      { "fuel": 1, "smoke": 2, "burn": 1, "volume_per_turn": "1250_ml" },
+      { "fuel": 1, "smoke": 1, "burn": 2 },
+      { "fuel": 1, "smoke": 1, "burn": 3 }
+    ]
+  },
+  {
+    "type": "material",
+    "ident": "cured_meat",
+    "name": "Cured Meat",
+    "density": 5,
+    "specific_heat_liquid": 3.16,
+    "specific_heat_solid": 2.31,
+    "latent_heat": 190,
+    "edible": true,
+    "bash_resist": 1,
+    "cut_resist": 1,
+    "acid_resist": 1,
+    "fire_resist": 1,
+    "elec_resist": 1,
+    "chip_resist": 2,
+    "dmg_adj": [ "bruised", "mutilated", "badly mutilated", "thoroughly mutilated" ],
+    "bash_dmg_verb": "bruised",
+    "cut_dmg_verb": "sliced",
+    "vitamins": [ [ "calcium", 0.1 ], [ "vitB", 1 ], [ "iron", 1.3 ] ],
+    "burn_data": [
+      { "fuel": 1, "smoke": 1, "burn": 1, "volume_per_turn": "2500_ml" },
+      { "fuel": 2, "smoke": 3, "burn": 2, "volume_per_turn": "10000_ml" },
+      { "fuel": 3, "smoke": 10, "burn": 3 }
+    ]
+  },
+  {
+    "type": "material",
+    "ident": "processed_food",
+    "name": "Processed fod",
+    "density": 4,
+    "specific_heat_liquid": 3,
+    "specific_heat_solid": 2,
+    "latent_heat": 190,    
+    "edible": true,
+    "bash_resist": 1,
+    "cut_resist": 1,
+    "acid_resist": 2,
+    "fire_resist": 1,
+    "elec_resist": 1,
+    "chip_resist": 2,
+    "dmg_adj": [ "lightly damaged", "damaged", "very damaged", "thoroughly damaged" ],
+    "bash_dmg_verb": "smashed",
+    "cut_dmg_verb": "cut",
+    "burn_data": [ { "fuel": 1, "smoke": 1, "burn": 1 }, { "fuel": 1, "smoke": 1, "burn": 2 }, { "fuel": 1, "smoke": 1, "burn": 3 } ]
+  },
+  {
+    "type": "material",
+    "ident": "cheese",
+    "name": "Cheese",
+    "density": 1,
+    "specific_heat_liquid": 3,
+    "specific_heat_solid": 2.7,
+    "latent_heat": 160,
+    "edible": true,
+    "bash_resist": 0,
+    "cut_resist": 0,
+    "acid_resist": 0,
+    "fire_resist": 0,
+    "elec_resist": 0,
+    "chip_resist": 0,
+    "dmg_adj": [ "lightly damaged", "damaged", "very damaged", "thoroughly damaged" ],
+    "bash_dmg_verb": "smashed",
+    "cut_dmg_verb": "cut",
+    "vitamins": [ [ "vitB", 0.5 ], [ "calcium", 1.5 ], [ "iron", 0.1 ] ],
```

Comment:
these can only be ints

Reply:
specific_heat_liquid and specific_heat_solid? They seem to be working fine in their current form.

Reply:
Pretty sure he means vitamins.

Reply:
The vitamin values were copy-pasted from already existing materials. Cheese for example from dairy.

Reply:
oh my apologies, with the large nutrition overhaul i forgot that materials themselves can have vitamins. maybe we should think about if that's a necessary thing anymore... but that's not really a thing for this PR